code/game/turfs/turf.dm

```diff
@@ -52,6 +52,16 @@
 	CalculateAdjacentTurfs()
 
 /turf/Destroy(force)
+	. = QDEL_HINT_IWILLGC
+	if(force)
+		..()
+		//this will completely wipe turf state
+		var/turf/basic/B = new /turf/basic(src)
```

Comment:
won't this break a heck load of shit @Cyberboss @MrStonedOne ?
(atmos adjacent turfs as a guess)